pkg/cloudprovider/providers/openstack/openstack.go

```diff
@@ -262,7 +262,7 @@ func getServerByName(client *gophercloud.ServiceClient, name string) (*servers.S
 	}
 
 	if len(serverList) == 0 {
-		return nil, ErrNotFound
+		return nil, cloudprovider.InstanceNotFound
```

Comment:
`ErrNotFound` is still referenced directly in a lot of places - it might be less confusing to either replace `ErrNotFound` entirely, or map to this cloudprovider error code just in `ExternalID`